### PR TITLE
Loadingfeed enhancing

### PR DESCRIPTION
**Pull request by jfversluis**
Added some simple logic to check if a feed is available and returns an a-ok!
With this the dummy variable seems redundant.

As per #169 love to hear your thoughts!

### CODE REVIEW

File: src/Firehose.Web/Infrastructure/CombinedFeedSource.cs [hunk 1]
```diff
@@ -61,7 +61,15 @@ private static ISyndicationFeedSource TryLoadFeed(IAmACommunityMember tamarin, U
...
                     .Grr();
-                var dummy = feedSource.Feed; // poke it to make sure it doesn't go bang.
+
+                // Poke it, to see if we can really reach it
+                var request = (HttpWebRequest)WebRequest.Create(uri);
+                request.Method = "HEAD";
+
+                var response = (HttpWebResponse)request.GetResponse();
+
+                if (response.StatusCode != HttpStatusCode.OK)
```
Comment (Cheesebaron) [inline src/Firehose.Web/Infrastructure/CombinedFeedSource.cs]: I think this should check status codes are between 200 and 399

Redirects are valid too, no?

Comment (jfversluis) [inline src/Firehose.Web/Infrastructure/CombinedFeedSource.cs]: Fair enough! Updated it

File: src/Firehose.Web/Infrastructure/CombinedFeedSource.cs [hunk 2]
```diff
@@ -61,7 +61,16 @@ private static ISyndicationFeedSource TryLoadFeed(IAmACommunityMember tamarin, U
...
+
+                // Poke it, to see if we can really reach it
+                var request = (HttpWebRequest)WebRequest.Create(uri);
+                request.Method = "HEAD";
+
+                var response = (HttpWebResponse)request.GetResponse();
+
+                // If return code is not success or redirect, it's no good
+                if ((int)response.StatusCode < 200 || (int)response.StatusCode > 399)
+                    throw new Exception("Feed says no");
```
Comment (jamesmontemagno) [inline src/Firehose.Web/Infrastructure/CombinedFeedSource.cs]: Is it okay just to throw an exception... will everything else just carry on?

Comment (jfversluis) [inline src/Firehose.Web/Infrastructure/CombinedFeedSource.cs]: It should be handled by the try/catch that is around it which was already in place when I got here. This logs some message to Elmah and returns null causing the feed to be excluded.

**Comment by jamesmontemagno** [comment]
I'm good with this.

**Pull request merged**